lib/platform.js

```diff
@@ -7,10 +7,11 @@ const env = process.env;
 module.exports = function () {
   const arch = env.npm_config_arch || process.arch;
   const platform = env.npm_config_platform || process.platform;
-  /* istanbul ignore next */
-  const libc = (platform === 'linux' && detectLibc.isNonGlibcLinuxSync()) ? detectLibc.familySync() : '';
+  const libc = process.env.LIBC || process.env.npm_config_libc || 
```

Comment:
Do we need to check both of these environment variables or would `env.npm_config_libc` suffice on its own?

Reply:
prebuild-install reads both LIBC-env value and npm_config_libc in this order. I made it this way to align with prebuild-install scripts, but if you want, LIBC-variable can be removed. I left it there mainly for backwards compatibility and their documentation no longer mentions it.

Reply:
After considering this, I removed reading of LIBC env value completely. Sharp installation has never supported it in the past and prebuild-install documentation now instructs the use of `--libc` or `npm_config_libc ` env value.